README.md

```diff
@@ -1,57 +1,35 @@
-# p-report-template
+# p-report
+[![Build Status](https://travis-ci.org/jsk-report-template/p-report.svg?branch=master)](https://travis-ci.org/jsk-report-template/p-report)
 
 Latex template for p-report
 
-[![Build Status](https://travis-ci.org/furushchev/p-report-template.svg)](https://travis-ci.org/furushchev/p-report-template)
-
 ### 1. Prerequisities
 
 ```bash
-# only for ubuntu 12.04
-$ sudo apt-add-repository ppa:texlive-backports/ppa
-$ sudo apt-get update
-$ sudo apt-get install lsb-release
+sudo apt-get update
+make install
 ```
 
 ### 2. Edit LaTeX files
 
 ### 3. Make pdf
 
 ```bash
-$ make
-```
-
-### 4. Release pdf
-
-- Install [Travis Command Line Tool](https://github.com/travis-ci/travis.rb#installation)
-- Enable Travis
-
-  ```bash
-  travis enable
-  travis setup releases --force
+make
+# or
+latexmk -pvc p-report
 ```
 
-- After `travis` command, configuration like below will be appended to `.travis.yml`.
+### 4. Make publication pdf
 
-  ```yaml
-deploy:
-  provider: releases
-  api_key: "GITHUB OAUTH TOKEN"
-  file: p-report.pdf
-  skip_cleanup: true
-  on:
-    tags: true
+```bash
+make pub
```

Comment:
I think writing like https://github.com/jsk-report-template/robomech-template#optional-convert-japanese-punctuations is kinder.
(Sorry that I forgot to fix this repository...)

Reply:
I updated.